toolkit/scripts/srpm_pack.mk

```diff
@@ -38,23 +38,27 @@ $(BUILD_SRPMS_DIR): $(STATUS_FLAGS_DIR)/build_srpms.flag
 	@echo Finished updating $@
 
 ifeq ($(DOWNLOAD_SRPMS),y)
+
+.SILENT: $(STATUS_FLAGS_DIR)/build_srpms.flag
+
 $(STATUS_FLAGS_DIR)/build_srpms.flag: $(local_specs) $(local_spec_dirs) $(SPECS_DIR)
 	for spec in $(local_specs); do \
 		spec_file=$${spec} && \
-		srpm_file=$$(rpmspec -q $${spec_file} --srpm --define='with_check 1' --define='dist $(DIST_TAG)' --queryformat %{NAME}-%{VERSION}-%{RELEASE}.src.rpm) && \
+		srpm_file=$$(rpmspec -q $${spec_file} --srpm --define='with_check 1' --define='dist $(DIST_TAG)' --queryformat %{NAME}-%{VERSION}-%{RELEASE}.src.rpm 2>/dev/null) && \
 		for url in $(SRPM_URL_LIST); do \
 			wget $${url}/$${srpm_file} \
```

Comment:
>wget [](http://example.com/codeflow?start=3&length=4)

To be consistent logging-wise, let's follow the example from `toolchain.mk`, so that `wget` prints its output to a log file, so that it can be printed in case of an error and available for debugging.